### ISSUE TITLE
update prompt in Altus- Both for whatsapp and Chrome.

### ISSUE THREAD

**Issue by emmanueljkm** [amanharwara/altus#194]
Hello,

I have just started using Altus. Thanks for your contribution in the realm of Open Source quality programs. I am being prompted from Altus to update both Chrome and Whatsapp. My version of altus is 4.71 on windows 10 21H2. What shall i do now? Do i need to update both? How if yes.

Thanks,

J Emmanuel

**Comment by amanharwara** [comment]
Could you share a screenshot of what you're referring to?

**Comment by duiveldoder** [comment]
Yes I have this too for some time now - not a biggy just restart the app once or twice and then it works

**Comment by emmanueljkm** [comment]
Sorry for the late response. And when i logout altus, before showing the initial QR code screen, there was also an intimation saying "Update Chrome". I could not take a screenshot on that  as it just splashes for less than 2/3 second and moving forward to that initial QR code screen. FYI.

Thank you and appreciated your reply.

With Warm Regards,

J Emmanuel
________________________________

**Comment by emmanueljkm** [comment]
My version info:

Altus: 4.7.1
  Electron: 18.0.4
  Chrome: 100.0.4896.75
  V8: 10.0.139.9-electron.0
  OS: Windows_NT x64 10.0.19044

**Comment by Johnnyy500** [comment]
> It shouldn't really cause any actual issues to your regular WhatsApp usage. Let me know if it does.
> Also, what version of Altus are you using?

It does actually. I have to add a new tab with the same number. Otherwise it doesn't work.

![image](https://user-images.githubusercontent.com/60765771/174759767-24c3bd6d-5ae1-4b42-92e9-80a341aacc9a.png)

How do I fix this? It keeps coming back. Thank you in advance!

**Comment by amanharwara** [comment]
> It causes issues with my regular use. I have to add a new tab with the same number. Otherwise, it doesn't work.

Hmm that's unexpected. Does reloading the page with `Cmd+Shift+R` not fix it?
